samples/angular/data/routes/styleguide/en.yml

```diff
@@ -2,253 +2,252 @@ fields:
-              sample: This text can be translated in en.yml
-      - componentName: StyleguideSection
-        fields:
-          heading: Angular Patterns
-        placeholders:
-          jss-styleguide-section:
-          - componentName: StyleguideAngularLazyLoading
-            fields:
-              heading: Lazy Loading
-
+              jss-styleguide-section:
+                - componentName: StyleguideSitecoreContext
+                  fields:
+                    heading: Sitecore Context
+                    description: <p><small>The Sitecore Context contains route-level data about the current context - for example, <code>pageState</code> enables conditionally executing code based on whether Sitecore is in Experience Editor or not.</small></p>
+                - componentName: StyleguideRouteFields
+                  fields:
+                    heading: Route-level Fields
+                    description: <p><small>Route-level content fields are defined on the <em>route</em> instead of on a <em>component</em>. This allows multiple components to share the field data on the same route - and querying is much easier on route level fields, making <em>custom route types</em> ideal for filterable/queryable data such as articles.</small></p>
+                - componentName: StyleguideComponentParams
+                  fields:
+                    heading: Component Params
+                    description: <p><small>Component params (also called Rendering Parameters) allow storing non-content parameters for a component. These params should be used for more technical options such as CSS class names or structural settings.</small></p>
+                  params:
+                    cssClass: alert alert-success
+                    # IMPORTANT: while params can be defined in the manifest as non-string types,
+                    # they are always sent to the component as strings.
+                    columns: 5
+                    useCallToAction: true
+                - componentName: StyleguideTracking
+                  fields:
+                    heading: Tracking
+                    description: <p><small>JSS supports tracking Sitecore analytics events from within apps. Give it a try with this handy interactive demo.</small></p>
+          - componentName: StyleguideSection
+            fields:
+              heading: Multilingual Patterns
+            placeholders:
+              jss-styleguide-section:
+                - componentName: StyleguideMultilingual
+                  fields:
+                    heading: Translation Patterns
+                    sample: This text can be translated in en.yml
+          - componentName: StyleguideSection
+            fields:
+              heading: Angular Patterns
+            placeholders:
+              jss-styleguide-section:
+                - componentName: StyleguideAngularLazyLoading
+                  fields:
+                    heading: Lazy Loading
```

Comment:
Looks like this one got reformatted, check your editor's tabsize, or install [the EditorConfig plugin for vscode](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) which our monorepo supports.